tests/test_echo.py

```diff
@@ -22,8 +32,10 @@ def _run_echo_haha(self,arch):
         # Need to dump by path because the program closes stdout
         assert pg.deadended[0].posix.stdout.concretize() == [b'haha\n']
 
-    def test_run_x86_64(self):
-        self._run_echo_haha('x86_64')
+    def test_echo_haha(self):
+        for arch in target_arches:
```

Comment:
It looks like you reverted some of the changes here when fixing a conflict

Reply:
It is still the same, I thought it looked clean to run it through iteration of versions, I will revert it back to how it was